test/libsolidity/syntaxTests/literals/unicode_string_direction_override_1.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function f() public pure
+    {
+        // pop 1
+        bytes memory s = unicode"underflow ‬";
+    }
+}
+// ----
+// ParserError 8936: (90-108): Unicode direction override underflow in comment.
```

Comment:
The error message mentions "comment" but the character is in the string. We should either have separate messages for this or at least make it more generic (e.g. `Unicode direction override underflow in comment or string literal.`)

Reply:
made messages to be more generic